pkgs/desktops/plasma-5/fetch.sh

```diff
@@ -1 +1 @@
-WGET_ARGS=( https://download.kde.org/stable/plasma/5.23.5/ -A '*.tar.xz' )
+WGET_ARGS=( https://download.kde.org/unstable/plasma/5.24.0/ -A '*.tar.xz' )
```

Comment:
I see I'm not the only one who forgot to change the path back to "stable"

Reply:
Ops!